pom.xml

```diff
@@ -0,0 +1,192 @@
+                    <artifactId>kotlin-stdlib-common</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>org.reactivestreams</groupId>
+            <artifactId>reactive-streams</artifactId>
+            <version>1.0.3</version>
+        </dependency>
+        <dependency>
+            <groupId>io.projectreactor</groupId>
+            <artifactId>reactor-core</artifactId>
+            <version>3.4.9</version>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.securevault</groupId>
+            <artifactId>org.wso2.securevault</artifactId>
+            <version>1.1.4</version>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon</groupId>
+            <artifactId>org.wso2.carbon.utils</artifactId>
+            <version>4.6.2</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>*</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>com.github.spotbugs</groupId>
+            <artifactId>spotbugs-annotations</artifactId>
+            <version>4.5.3</version>
+        </dependency>
+    </dependencies>
+
+    <packaging>bundle</packaging>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>5.1.4</version>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Export-Package>
+                            org.wso2.carbon.securevault.azure
+                        </Export-Package>
+                        <DynamicImport-Package>*</DynamicImport-Package>
+                        <Import-Package>
+                            org.wso2.securevault.*;version="[1.1.0, 2.0.0)"
+                        </Import-Package>
+                        <Embed-Dependency>
+                            azure-core;
+                            azure-core-http-okhttp;
+                            azure-identity;
+                            azure-security-keyvault-secrets;
+                            content-type;
+                            jackson-dataformat-xml;
+                            jackson-datatype-jsr310;
+                            kotlin-stdlib;
+                            msal4j;
+                            oauth2-oidc-sdk;
+                            okhttp;
+                            okio;
+                            reactive-streams;
+                            reactor-core
+                        </Embed-Dependency>
+                        <_fixupmessages>"Classes found in the wrong directory..."</_fixupmessages>
```

Comment:
what is the use case of this?

Reply:
Some of the embedded dependencies are multi-release jars that support Java 9, but the maven bundle plugin used to build the OSGi bundle doesn't support multi-release jars yet and gives the mentioned warning This is meant to address that.